examples/MLIRGPU/makefile

```diff
@@ -69,5 +69,14 @@ gpu-mma-run:
 	${MLIR_CPU_RUNNER} -entry-point-result=void -shared-libs=${MLIR_CUDA_RUNTIME} -shared-libs=${MLIR_RUNNER_UTILS}
 
 vector-to-gpu-lower:
-	@${MLIR_OPT} vector-to-gpu.mlir -gpu-kernel-outlining | \
-	${MLIR_OPT} -pass-pipeline="builtin.module(func.func(convert-vector-to-gpu,canonicalize))" -o log.mlir 
+	@${MLIR_OPT} vector-to-gpu.mlir -convert-linalg-to-loops | \
+	${MLIR_OPT} -pass-pipeline="builtin.module(gpu.module(convert-vector-to-gpu,canonicalize))" | \
+	${MLIR_OPT} -pass-pipeline='builtin.module(gpu.module(strip-debuginfo,convert-gpu-to-nvvm,gpu-to-cubin{chip=sm_70}))' | \
+	${MLIR_OPT}	-convert-scf-to-cf -gpu-to-llvm -o log.mlir 
+
+vector-to-gpu-run:
```

Comment:
It looks like the bugs still haven't been fixed, the `vector-to-gpu` example still won't run, and the `gpu-all-reduce-and`.